arch/arm/core/aarch64/isr_wrapper.S

```diff
@@ -83,8 +82,6 @@ spurious_continue:
 
 	/* --(_kernel->nested) */
 	dec_nest_counter x0, x1
-
-	cmp	x1, #0
 	bne	exit
```

Comment:
Nit: I'd tend to prefer leaving `dec_nest_counter` as-is and changing these to `cbnz` instead. Same number of instructions and doesn't rely on carrying flags across (macro, admittedly) calls.

...though now that I look at it clang appears to prefer `subs` / `bne` here: https://godbolt.org/z/vrqzdd